verticapy/learn/vmodel.py

```diff
@@ -269,16 +269,9 @@ def drop(self):
 	---------------------------------------------------------------------------
 	Drops the model from the Vertica database.
 		"""
-        if self.type == "AutoDataPrep":
-            with warnings.catch_warnings(record=True) as w:
-                drop(
-                    self.name, self.cursor, method="table",
-                )
-        else:
-            with warnings.catch_warnings(record=True) as w:
-                drop(
-                    self.name, self.cursor, method="model",
-                )
+        with warnings.catch_warnings(record=True) as w:
+            drop(self.name)
+            drop(self.name)
```

Comment:
These 2 lines are the same! Have you forgotten to specify the method?

Reply:
I just updated this in my last commit